Update readme.md with SetUp process for new contributers (#238)
DeXter-on-Radix/website: CONTRIBUTING.md

- **Write tests** to prevent regressions, unit tests with [Jest](https://jestjs.io/) in `__tests__` folder, end to end tests with [Playwright](https://playwright.dev/) in `e2e` folder

If you happen to use VS Code, install the recommended extensions to get automatic formatting and linting on save, they are listed in `.vscode/extensions.json` and VS Code will prompt you to install them.

> In order to contribute, you need to set up the development envirment. 
> Use the following set up section for instructions.

Reviewer: ```suggestion
```

I think we can remove it. The "Set up" title says it all. If not removing then fixing the word envirment to environment.

> <h2> Set Up </h2>

Reviewer: ```suggestion
## Set Up
```

- Fork the Dexter repository to your github account.
- Clone from your forked repo.
- Make a new branch with a feature name (you can see some example in [closed PRs](https://github.com/DeXter-on-Radix/website/pulls?q=is%3Apr+is%3Aclosed)).
- Make some changes.
- Commit and push local branch changes to your own repo.
- Make a pull request into the [original Dexter repo](https://github.com/DeXter-on-Radix/website).

> Now you are ready to run the project locally.(instructions to be found in README.md ).

Reviewer: ```suggestion
Now you are ready to [run the project locally](/README.md#Getting-Started).
```

Author: As you mentioned that running locally an contributing not necaseraly related to each other, therefore would discard this change.
